paasta_tools/cli/cmds/local_run.py

```diff
@@ -418,15 +419,17 @@ def docker_pull_image(docker_url):
     bindings due to the docker auth/registry transition. Once we are past Docker 1.6
     we can use better credential management, but for now this function assumes the
     user running the command has already been authorized for the registry"""
-    paasta_print("Please wait while the image (%s) is pulled..." % docker_url, file=sys.stderr)
+    paasta_print("Please wait while the image (%s) is pulled (times out after 30m)..." % docker_url, file=sys.stderr)
```

Comment:
Can you move the timeout to a variable that you reference here and in the call to `timed_flock`? That way, they stay in sync even when we update the timeout in the future.